common/static/common/js/discussion/views/discussion_thread_profile_view.js

```diff
@@ -42,31 +41,27 @@
                         }
                     });
                 }
-                this.$el.html(_.template($("#profile-thread-template").html())(params));
-                this.$("span.timeago").timeago();
-                element = this.$(".post-body");
-                if (typeof MathJax !== "undefined" && MathJax !== null) {
-                    MathJax.Hub.Queue(["Typeset", MathJax.Hub, element[0]]);
-                }
+                edx.HtmlUtils.setHtml(
+                    this.$el,
+                    edx.HtmlUtils.template($('#profile-thread-template').html())(params)
+                );
+                this.$('span.timeago').timeago();
+                DiscussionUtil.typesetMathJax(this.$('.post-body'));
                 return this;
             };
 
             DiscussionThreadProfileView.prototype.convertMath = function() {
-                return this.model.set(
-                    'markdownBody',
-                    DiscussionUtil.postMathJaxProcessor(DiscussionUtil.markdownWithHighlight(this.model.get('body')))
-                );
+                var htmlSnippet = DiscussionUtil.markdownWithHighlight(this.model.get('body'));
+                this.model.set('markdownBody', htmlSnippet);
```

Comment:
It totally is, otherwise MathJax syntax isn't rendered on the user profile page!  Thanks for catching that.  Fixed with 89ce37f.